appveyor.yml

```diff
@@ -76,7 +76,7 @@ test_script:
             Push-Location $testFolder
 
             Write-Host "[$env:configuration] Running tests in $testFolder" -foregroundcolor "magenta"
-            dotnet test -c $env:configuration
+            dotnet test -c $env:configuration --no-build
```

Comment:
What does this do?

Reply:
@fassadlr it prevents rebuilding the dlls you just built :-), altogether gaining a few seconds ;-)